src/components/Entities/EntityVersions.tsx

```diff
@@ -62,7 +62,7 @@ export const EntityVersions: React.FC<EntityVersionsProps> = ({
         { domain, project },
         {
             sort,
-            filter: [...baseFilters, ...filtersState.appliedFilters],
+            filter: baseFilters,
```

Comment:
I see there is three filters which are using `[...baseFilters, ...filtersState.appliedFilters],` currently in code. Can you explain why filterState.appliedFilters is not needed here?
Also if it is removed, than it could also be removed from line 50 and line 82 in this functions, as they become redundant.

Reply:
If the filter is not needed than filtersState mentions in line 50 and line 82 can also be removed 🤔

Reply:
@anrusina That's a good catch. Just pushed fix!